apps/taskman/service.py

```diff
@@ -174,6 +174,8 @@ def _generate_task_data(self, flaw: Flaw):
         if flaw.major_incident_state in [
             Flaw.FlawMajorIncident.APPROVED,
             Flaw.FlawMajorIncident.CISA_APPROVED,
+            Flaw.FlawMajorIncident.MINOR,
```

Comment:
Good point. I was also a bit hesitant here. So far we used only a single label and it is probably only our initiative to set it in this case as do not think people care for it in the OSIM tasks. As it is internal for PSIRT (not engineers) however and they own those tasks I at the end decided to put it there.

I am not sure how to approach this label. We can have different labels for different incident types. We can omit minor there. We can just leave it as it is until someone cares. Do you have a preference? I think I do not :smile: 

The whole confusing thing is now that we have a variable **major_incident_state** and it can actually hold an information saying that we have a **minor incident**. The rename would be ideal but laborious.